### PR TITLE
Switch to Debian sid and Qemu 3.0

### PR DESCRIPTION
**Pull request by shenki**
These changes are in preparation for enabling ppc64le.

### CODE REVIEW

File: debian-setup.sh [hunk 1]
```diff
@@ -25,10 +25,18 @@ apt-get install -y -qq \
...
 
+
+# Install Qemu 3.0 from shenki's ppa. Debian nor Ubuntu package 3.0 yet, which
+# is required for ppc64le powernv support
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E007EC6A
+echo "deb http://ppa.launchpad.net/shenki/ppa/ubuntu bionic main" | tee -a /etc/apt/sources.list
+apt-get update -qq
+apt-get install -y -qq \
+    qemu-system-arm \
+    qemu-system-x86
```
Comment (nickdesaulniers) [inline debian-setup.sh]: please redirect stdout to /dev/null as in L28 and L47. 😯

File: .travis.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ cache:
 services:
   - docker
 script:
-  - docker create -v ${TRAVIS_BUILD_DIR}:/travis --name travis debian
-  - docker run --env ARCH=${ARCH} --env LD=${LD} --rm --volumes-from travis debian /bin/bash -c 'cd /travis && ./debian-setup.sh && ./driver.sh && ccache -s'
+  - docker create -v ${TRAVIS_BUILD_DIR}:/travis --name travis debian:sid
+  - docker run --env ARCH=${ARCH} --env LD=${LD} --rm --volumes-from travis debian:sid /bin/bash -c 'cd /travis && ./debian-setup.sh && ./driver.sh && ccache -s'
```
Comment (nickdesaulniers) [inline .travis.yml]: What's the default if you don't specify?

**Comment by nickdesaulniers** [comment]
Cool! Thanks for the patches! 🏎

File: .travis.yml (hunk 2 above)
Comment (shenki) [reply .travis.yml]: Stable, I think:
```
$ docker run -it debian cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
```

File: debian-setup.sh (hunk 1 above)
Comment (shenki) [reply debian-setup.sh]: :+1:

File: .travis.yml (hunk 2 above)
Comment (nathanchance) [reply .travis.yml]: Yeah, it's just a Stretch image by default.

File: debian-setup.sh [hunk 3]
```diff
@@ -25,10 +25,18 @@ apt-get install -y -qq \
     libssl-dev \
     make \
     openssl \
-    qemu-system-arm \
-    qemu-system-x86 \
     >/dev/null
 
+
```
Comment (nathanchance) [inline debian-setup.sh]: I think this extra line should go (or two spaces should be added in other places).

**Pull request merged**